File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -205,7 +205,8 @@ The .NET PowerPoint Library (Presentation) allows you to convert PowerPoint slid
 For instance, if a slide includes bulleted paragraphs, each having entrance animation effects, the Presentation library converts every bulleted paragraph into a separate image.
 
 N> 1. Only entrance animation effects are considered for generating separate images. Other animation effects and non-animated elements will be converted into images within the first image itself.
-N> 2. Converting PowerPoint animations to images is not supported in the UWP platform. 
+N> 2. Converting PowerPoint animations to images is not supported in the UWP platform.
```

Comment:
N> 2. The UWP platform does not support the conversion of PowerPoint animations to images.